src/Tabs.js

```diff
@@ -20,6 +20,8 @@ const propTypes = {
    * @controllable onSelect
    */
   activeKey: PropTypes.any,
+  /** The default active key that is expanded on start */
```

Comment:
```suggestion
  /** The default active key that is selected on start */
```

as this is not an accordion